source/lib-intro.tex

```diff
@@ -1061,55 +1061,44 @@
 
 
 \pnum
-The facilities of the C standard library are provided in 26
-\indextext{library!C standard}%
-additional headers, as shown in Table~\ref{tab:cpp.c.headers}.%
+The facilities of the C standard library\indextext{library!C standard}
+are provided in 22 additional headers, as shown in Table~\ref{tab:cpp.c.headers}.%
```

Comment:
Happy to go with whatever you decide, just calling it out so that we make a deliberate decision, rather than accidentally fall into the old pattern by inertia.